### PR TITLE
Fix warning message mentioning quoted atoms may end with ! or ?

### PR DESCRIPTION
**Pull request by eksperimental**
Closes https://github.com/elixir-lang/elixir/issues/10722

### CODE REVIEW

File: lib/elixir/src/elixir_tokenizer.erl [hunk 1]
```diff
@@ -438,7 +438,7 @@ tokenize([$:, H | T] = Original, Line, Column, Scope, Tokens) when ?is_quote(H)
         true ->
           WarnMsg = io_lib:format(
             "found quoted atom \"~ts\" but the quotes are not required. "
-            "Atoms made exclusively of ASCII letters, numbers, and underscores "
+            "Atoms made exclusively of ASCII letters, numbers, and underscores which may optionally end with ! or ?"
```
Comment (josevalim) [inline lib/elixir/src/elixir_tokenizer.erl:441]:
```suggestion
            "Atoms made exclusively of ASCII letters, numbers, and underscores, which may optionally end with ! or ?, "
```

?

Comment (josevalim) [reply lib/elixir/src/elixir_tokenizer.erl:441]: Or:

```suggestion
            "Atoms made exclusively of ASCII letters, numbers, underscores, and optionally end with ! or ? "
```

Comment (michallepicki) [inline lib/elixir/src/elixir_tokenizer.erl:441]: I like the second option more but I think it needs `ing` in `ending` to form a valid sentence, like this:
```suggestion
            "Atoms made exclusively of ASCII letters, numbers, and underscores, and optionally ending with ! or ? "
```

Comment (eksperimental) [reply lib/elixir/src/elixir_tokenizer.erl:441]: Correct. I was about to suggest the same thing.

**Comment by eksperimental** [comment]
I guess this is ready to be merged.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**